### DISCUSSION TITLE
Talk Python to Me

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#135] [General]
I'll be discussing Piccolo with Michael Kennedy on the [Talk Python to Me](https://talkpython.fm/) podcast on 22nd July at roughly 10pm UK time.

There will be a live stream here:

https://www.youtube.com/watch?v=d3WF59IO3S0

And the podcast will become available at a later date.

I'll be covering the origins of Piccolo, why I think it's cool, and ideas for the future.

If anyone has anything particular they want me to cover, let me know :)

**Comment by heliumbrain** [comment]
That's really cool! I like his podcast.

I think it can be a good idea to give a intro to the ASGI spec. Especially how it all ties together, the composability of it and how Piccolo adheres to it.

**Reply by dantownsend** [reply]
Good idea - will make sure I mention this

**Comment by dantownsend** [comment]
It's on YouTube now: https://www.youtube.com/watch?v=d3WF59IO3S0

Podcast should come out in a couple of weeks.
